lib/models/Cats.js

```diff
@@ -0,0 +1,59 @@
+const pool = require('../utils/pool');
+
+module.exports = class Cat {
+  id;
+  name;
+  favorite_treat;
+  favorite_toy;
+
+  constructor(row) {
+    this.id = row.id;
+    this.name = row.name;
+    this.favorite_treat = row.favorite_treat;
```

Comment:
this is where you would use camel case `this.favoriteTreat = row.favorite_treat`